code/mod_table/mod_table.cpp

```diff
@@ -211,6 +212,17 @@ void parse_mod_table(const char *filename)
 				 mprintf(("Game Settings Table: Setting shield pain flash factor to %.2f\n", Shield_pain_flash_factor));
 		}
 
+		if (optional_string("$BMPMAN Slot Limit:")) {
+			int slots;
+			stuff_int(&slots);
+			if (slots < 3500) {
+				Warning(LOCATION, "Invalid BMPMAN slot limit [%d]; must be at least 3500.\n", slots);
```

Comment:
This should use `error_display` so that the warning message also includes the file and line number of the currently parsed table.

Reply:
None of the other parse warnings in mod_table.cpp use `error_display()`; I was just keeping consistent.

Reply:
In my opinion, it is more important to use the right function for the warnings instead of keeping the current style of the file.